lib/crex.js

```diff
@@ -39,7 +41,7 @@ CrEx.prototype.getUrl = function () {
 
 CrEx.prototype.getAddress = function () {
 	var url = this.url;
-	return this.port !== '' ? url + ':' + this.port : url;
+	return (this.port !== '' ? url + ':' + this.port : url) + (this.proxy ? ' (proxy: ' + this.proxy + ')' : '');
```

Comment:
Saw that this function is only used for console purposes. That's why I concat the proxy info.